bin/cli.js

```diff
@@ -40,6 +41,10 @@ const browserLaunchers = {
   'Safari':    'karma-safari-launcher'
 };
 
+const logLevels = Object.keys(karmaLogLevels);
```

Comment:
Could you provide this mapping manually here instead of relying on the key order provided by `Object.keys`.
Also, I thought we didn't support having `disable` log level?

Reply:
I'll leave it for `karmaLogLevel`, but remove from verbosity. Is it ok?

Reply:
I think we can have it as `quiet` option - no output at all.

Reply:
For now, with no verbosity options provided `loglevel` will be `error` as you suggested [here](https://github.com/metarhia/metatests/pull/94#discussion_r221919699).

Reply:
And as for `quiet` - in this case we need to either complete `reporter` to support such functionality, though this won't prevent output from code itself or to run tests in separate process and get `stdout` and `stderror` from it, but I'm not a fan of this idea.

Reply:
PTAL https://github.com/metarhia/metatests/pull/99, you can now set `reporter` to null and there will be no output from `metatests` runner.